plugin/src/main/groovy/com/btkelly/gnag/reporters/CheckstyleViolationDetector.groovy

```diff
@@ -46,38 +47,18 @@ class CheckstyleViolationDetector extends BaseExecutedViolationDetector {
             checkStyleTask.setConfigUrl(getClass().getClassLoader().getResource("checkstyle.xml"))
         }
 
-        projectHelper.getSources().findAll { it.exists() }.each {
-            checkStyleTask.addFileset(project.ant.fileset(dir: it))
+        projectHelper.getJavaSourceFiles().each { sourceFile ->
```

Comment:
Everything in these reporter methods is now based on individual files, rather than source set directories. This is to handle mixed file types in the same source set directory.